gas-station/typescript/src/sponsored_examples.ts

```diff
@@ -13,16 +9,16 @@ const SENDER_PRIVATE_KEY = "<PRIVATE_KEY>";
 const SENDER_RECOVERY_PHRASE = "<RECOVERY_PHRASE>";
 
 // gas budget, in MIST
-const GAS_BUDGET = 5000000;    
+const GAS_BUDGET = 5000000;
 
 // Gas Station endpoint:
 const SPONSOR_RPC_URL = "https://api.shinami.com/gas/v1/<GAS_ACCESS_KEY>";
 
 // Sui Node endpoint:
-const connection = new Connection({
-    fullnode: 'https://node.shinami.com/api/v1/<NODE_ACCESS_KEY>'
+const suiClient = new SuiClient({
+  // Sui Node endpoint:
+  url: "https://node.shinami.com/api/v1/<NODE_ACCESS_KEY>",
```

Comment:
Our own docs are a bit out of date 😅
Can you change this to `https://api.shinami.com/node/v1/<NODE_ACCESS_KEY>`?

Reply:
Yep, done!